app/redhat_insights/default/data/ui/views/main_dashboard.xml

```diff
@@ -1051,6 +1110,54 @@
 					}
 				}
 			},
+			{
+				"item": "viz_RDVKOOtq",
+				"type": "line",
+				"position": {
+					"from": {
+						"x": 665,
+						"y": 217
+					},
+					"to": {
+						"x": 1170,
+						"y": 219
+					}
+				}
+			},
+			{
+				"item": "viz_lQdKYbSx",
+				"type": "line",
+				"position": {
+					"from": {
+						"x": 662,
+						"y": 272
+					},
+					"to": {
+						"x": 1167,
+						"y": 274
+					}
+				}
+			},
+			{
+				"item": "viz_G743tctf",
+				"type": "block",
+				"position": {
+					"x": 650,
+					"y": 0,
+					"w": 540,
+					"h": 140
+				}
+			},
+			{
+				"item": "viz_89oyYK2v",
+				"type": "block",
+				"position": {
```

Comment:
The value is weirdly aligned:
![Screenshot 2022-09-02 at 15-44-35 Dashboard](https://user-images.githubusercontent.com/7695766/188159666-03d42f2e-dd6a-43c3-b6fe-7c4fe690179f.png)

Reply:
All these objects are center-aligned. So, depending to the number and arrow, will be change on alignment, see Patch for example.

Reply:
Fixed:
![image](https://user-images.githubusercontent.com/2904206/188165560-57bf1408-4ae3-46e3-b037-ce0a6d362dfb.png)